doc/reference/storage_btrfs.md

```diff
@@ -55,6 +55,7 @@ The following configuration options are available for storage pools that use the
 Key                             | Type      | Default                    | Description
 :--                             | :---      | :------                    | :----------
 btrfs.mount\_options            | string    | user\_subvol\_rm\_allowed  | Mount options for block devices
+size                            | string    | 0                          | Size of the storage pool when creating loop-based pools (in bytes, suffixes supported)
```

Comment:
We should probably update the default value here to a link indicating how we calculate the size of a loop file if size isn't specified. Which according to the source here https://github.com/lxc/lxd/blob/master/lxd/storage/drivers/utils.go#L882-L891 is:

20% of free disk space, up to 30GB, or at least 5GB.